frontend/src/components/AlbumCard.vue

```diff
@@ -1,3 +1,365 @@
+        >
+          +{{ imageUrls.length - 4 }}
+        </div>
+      </div>
+      <div v-else :class="$style.noImage">
+        <div :class="$style.noImageIcon" aria-hidden="true">📷</div>
+        <div :class="$style.noImageText">No Images</div>
+      </div>
+    </div>
+    <div :class="$style.content">
+      <h3 :class="$style.title">{{ title }}</h3>
+      <div :class="$style.meta">
+        <div :class="$style.leftMeta">
+          <div :class="$style.creator">
+            <img
+              v-if="creator.avatarUrl"
+              :src="creator.avatarUrl"
+              :alt="`${creator.name}'s avatar`"
+              :class="$style.creatorAvatar"
+              @error="handleAvatarError"
+            />
+            <div v-else :class="$style.creatorAvatarPlaceholder">
+              {{ creator.name.charAt(0).toUpperCase() }}
+            </div>
+            <span :class="$style.creatorName">{{ creator.name }}</span>
+          </div>
+        </div>
+        <div :class="$style.rightMeta">
+          <span :class="$style.imageCount">{{ imageUrls.length }} images</span>
+          <span :class="$style.date">{{ formatDate(createdAt) }}</span>
+        </div>
+      </div>
+    </div>
+  </div>
 </template>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+
+interface Props {
+  id: string
+  title: string
+  description?: string // オプショナルに変更
+  imageUrls: string[]
+  createdAt: string
+  creator: {
+    id: string
+    name: string
+    avatarUrl?: string
+  }
+}
+
+const props = defineProps<Props>()
+
+// 表示する画像（最大4枚）
+const displayImages = computed(() => props.imageUrls.slice(0, 4))
+
+// 画像読み込みエラー時の処理
+const handleImageError = (event: Event) => {
+  const img = event.target as HTMLImageElement
+  if (img) {
+    img.src = '/dummyAlbumsIcon.png' // フォールバック画像
+  }
+}
+
+// アバター読み込みエラー時の処理
+const handleAvatarError = (event: Event) => {
+  const img = event.target as HTMLImageElement
+  if (img) {
+    // アバターエラー時は非表示にして、プレースホルダーを表示
+    img.style.display = 'none'
+  }
+}
+
+// 画像アイテムのクラスを決定する関数
+const getImageItemClass = (totalImages: number, index: number): string => {
+  const count = Math.min(totalImages, 4)
+
+  if (count === 1) return 'single'
+  if (count === 2) return 'half'
+  if (count === 3) {
+    return index === 0 ? 'thirdMain' : 'thirdSub'
+  }
+  return 'quarter'
+}
+
+// 日付をフォーマットする関数
+const formatDate = (dateString: string): string => {
+  const date = new Date(dateString)
+  return date.toLocaleDateString('ja-JP', {
+    year: 'numeric',
+    month: 'short',
+    day: 'numeric',
+  })
+}
+</script>
+
+<style lang="scss" module>
+.albumCard {
+  background: white;
+  border-radius: 8px;
+  border: 1px solid var(--Light-UI-Tertiary, #ced6db);
+  overflow: hidden;
+  transition: all 0.15s cubic-bezier(0.4, 0, 0.2, 1);
+  cursor: pointer;
+  max-width: 300px;
+  user-select: none;
+  display: flex;
+  flex-direction: column;
+  height: 330px; /* 固定高さを設定 */
```

Comment:
ここも開発時のコメント残ってる。
意図がない限りは、ちょっと冗長かも。

```suggestion
  height: 330px;
```

Reply:
[9785418](https://github.com/traP-jp/1m25_10/pull/130/commits/9785418da0ba9aa2d9cb279c1f58186c6d1a8307)